### PR TITLE
Aleksei/sentry add arch to link

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

Fixed arch value replaced with arch result

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
did this fix it?

File: tasks/sentry.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # install sentry cli
 echo "Installing and configuring sentry cli"
-curl -L -o sentry-cli https://github.com/getsentry/sentry-cli/releases/download/1.40.0/sentry-cli-Linux-x86_64
+curl -L -o sentry-cli https://github.com/getsentry/sentry-cli/releases/download/1.40.0/sentry-cli-Linux-`arch`
```
Comment (Bitcoinera) [inline tasks/sentry.sh]: Why does it need to be a string like that for `arch` and not for `x86_64`?

Comment (iambeone) [reply tasks/sentry.sh]: It's not just a string, it's a command. Placing it in ` means that we need to put here its result

Comment (Bitcoinera) [reply tasks/sentry.sh]: oh I see. And what does the command `arch` then do? Fetch the `arch` version?

Comment (iambeone) [reply tasks/sentry.sh]: Yap, it prints computer architecture.

Comment (Bitcoinera) [reply tasks/sentry.sh]: Nice!

**Pull request merged**